### PR TITLE
Refactor Get and update headers for digital Twin

### PR DESCRIPTION
**Pull request by bikamani**
Refactor Get and update headers for digital Twin

### CODE REVIEW

File: service/iot-service-client/code-model-v1 [hunk 1]
```diff
@@ -24,11 +24,11 @@
         "2020-09-30"
        ],
        "filename": [
-        "mem:///103?oai3.shaken.json"
+        "mem:///102?oai3.shaken.json"
```
Comment (bikamani) [inline service/iot-service-client/code-model-v1:27]: I am taking care of this in gitignore in another PR

File: iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java [hunk 2]
```diff
@@ -1,33 +1,24 @@
 package tests.integration.com.microsoft.azure.sdk.iot.digitaltwin;
```
Comment (vinagesh) [inline iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java:1]: Copyright

Comment (vinagesh) [reply iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java:1]: Might be needed for the other test file too.

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java [hunk 3]
```diff
@@ -85,7 +82,7 @@ public static DigitalTwinAsyncClient createFromConnectionString(String connectio
      * @param digitalTwinId The Id of the digital twin.
      * @param clazz The class to deserialize the application/json into.
      * @param <T> The generic type to deserialize the application/json into.
-     * @return A {@link ServiceResponseWithHeaders} representing deserialized application/json of the digital twin with {@link DigitalTwinGetHeaders}.
+     * @return A {@link ServiceResponseWithHeaders} representing deserialized application/json of the digital twin with {@link DigitalTwinGetDigitalTwinHeaders}.
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java:85]: fix return type header

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java [hunk 4]
```diff
@@ -123,7 +121,7 @@ public static DigitalTwinAsyncClient createFromConnectionString(String connectio
      * Updates a digital twin.
      * @param digitalTwinId The Id of the digital twin.
      * @param digitalTwinUpdateOperations The JSON patch to apply to the specified digital twin. This argument can be created using {@link UpdateOperationUtility}.
-     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateHeaders}.
+     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateDigitalTwinHeaders}.
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java:124]: same

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java [hunk 5]
```diff
@@ -135,12 +133,13 @@ public static DigitalTwinAsyncClient createFromConnectionString(String connectio
      * @param digitalTwinId The Id of the digital twin.
      * @param digitalTwinUpdateOperations The JSON patch to apply to the specified digital twin. This argument can be created using {@link UpdateOperationUtility}.
      * @param options The optional settings for this request.
-     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateHeaders}.
+     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateDigitalTwinHeaders}.
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java:136]: same

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinClient.java [hunk 6]
```diff
@@ -50,7 +47,7 @@ public static DigitalTwinClient createFromConnectionString(String connectionStri
      * @param digitalTwinId The Id of the digital twin.
      * @param clazz The class to deserialize the application/json into.
      * @param <T> The generic type to deserialize the application/json into.
-     * @return A {@link ServiceResponseWithHeaders} representing deserialized application/json of the digital twin with {@link DigitalTwinGetHeaders}.
+     * @return A {@link ServiceResponseWithHeaders} representing deserialized application/json of the digital twin with {@link DigitalTwinGetDigitalTwinHeaders}.
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinClient.java:50]: same

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinClient.java [hunk 7]
```diff
@@ -74,7 +71,7 @@ public Void updateDigitalTwin (String digitalTwinId, List<Object> digitalTwinUpd
      * Updates a digital twin.
      * @param digitalTwinId The Id of the digital twin.
      * @param digitalTwinUpdateOperations The JSON patch to apply to the specified digital twin. This argument can be created using {@link UpdateOperationUtility}.
-     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateHeaders}.
+     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateDigitalTwinHeaders}.
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinClient.java:74]: same

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinClient.java [hunk 8]
```diff
@@ -86,7 +83,7 @@ public Void updateDigitalTwin (String digitalTwinId, List<Object> digitalTwinUpd
      * @param digitalTwinId The Id of the digital twin.
      * @param digitalTwinUpdateOperations The JSON patch to apply to the specified digital twin. This argument can be created using {@link UpdateOperationUtility}.
      * @param options The optional settings for this request.
-     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateHeaders}.
+     * @return A {@link ServiceResponseWithHeaders} with {@link DigitalTwinUpdateDigitalTwinHeaders}.
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinClient.java:86]: same

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/models/DigitalTwinGetDigitalTwinHeaders.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
+/**
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/models/DigitalTwinGetDigitalTwinHeaders.java:1]: Fix Copyright

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/models/DigitalTwinUpdateDigitalTwinHeaders.java [hunk 10]
```diff
@@ -0,0 +1,67 @@
+/**
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/models/DigitalTwinUpdateDigitalTwinHeaders.java:1]: Fix Copyright

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinGetHeaders.java [hunk 11]
```diff
@@ -4,7 +4,7 @@
  * regenerated.
  */
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinGetHeaders.java:6]: Add Copyright - no longer generated

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinUpdateHeaders.java [hunk 12]
```diff
@@ -4,7 +4,7 @@
  * regenerated.
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinUpdateHeaders.java:4]: Add Copyright - no longer generated

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/swagger/digitalTwin.json [hunk 13]
```diff
@@ -1,270 +1,272 @@
...
-          }
-        ],
-        "responses": {
-          "202": {
-            "description": "Accepted",
-            "headers": {
-              "ETag": {
-                "description": "Weak Etag of the modified resource",
-                "type": "string"
+    "swagger":  "2.0",
```
Comment (vinagesh) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/swagger/digitalTwin.json:2]: formatting is lost

**Review by vinagesh — APPROVED** [review]

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java (hunk 3 above)
Comment (bikamani) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java:85]: Good catch

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinGetHeaders.java (hunk 11 above)
Comment (bikamani) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinGetHeaders.java:6]: done

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/swagger/digitalTwin.json (hunk 13 above)
Comment (bikamani) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/swagger/digitalTwin.json]: done

**Pull request merged**